packages/terra-clinical-result/docs/FlowsheetResultCell.md

```diff
@@ -0,0 +1,182 @@
+# Terra Clinical Flowsheet Result Cell
+
+The Clinical Result is a standardized view for single result values added to a Patient's Medical Chart. The flowsheet result cell is specifically for the grid display of a result collection, and can handle displaying indicaiton of multiple results.
```

Comment:
typo: `indicaiton`

I recommend this plugin: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker. It's definitely cut down on my typo output.

Reply:
Added plugin to VSCode, and Resolved https://github.com/cerner/terra-clinical/pull/574/commits/95ff8c13f61df4d9a218f6d91a9237404301ff3e